examples/motoko_examples/http_counter/src/main.ts

```diff
@@ -0,0 +1,164 @@
+import { Update, Opt, nat16, Variant, Query, nat8, Func, ic } from 'azle';
+import encodeUtf8 from 'encode-utf8';
+
+type StreamingCallbackHttpResponse = {
+    body: nat8[];
+    token: Opt<Token>;
+};
+
+type Token = {
+    // add whatever fields you'd like
+    arbitrary_data: string;
+};
+
+type CallbackStrategy = {
+    callback: Callback;
+    token: Token;
+};
+
+type Callback = Func<(t: Token) => Query<StreamingCallbackHttpResponse>>;
+
+type StreamingStrategy = Variant<{
+    Callback: CallbackStrategy;
+}>;
+
+type HeaderField = [string, string];
+
+type HttpResponse = {
+    status_code: nat16;
+    headers: HeaderField[];
+    body: nat8[];
+    streaming_strategy: Opt<StreamingStrategy>;
+    upgrade: Opt<boolean>;
+};
+
+type HttpRequest = {
+    method: string;
+    url: string;
+    headers: HeaderField[];
+    body: nat8[];
+};
+
+// Look at the stable examples
+// TODO what is the stable keyword?
+let counter = 0;
```

Comment:
Did we decide not to implement stable storage here? If we didn't, then let's do so